### PR TITLE
Default card

### PR DESCRIPTION
**Pull request by TobySalusky**
* index for overviews
* selectedOverviews added to cont3xt user config
* UI for default card
* UI for overview configuration
* create blank default overviews for each itype

### CODE REVIEW

File: cont3xt/vueapp/src/components/pages/Settings.vue [hunk 1]
```diff
@@ -28,6 +28,39 @@
...
+              class="float-right"
+              variant="secondary"
+              v-if="visibleTab === 'overviews'"
+              @click.stop.prevent="openOverviewForm"
+              v-b-tooltip.hover="'Create a new overview'">
+            <span class="fa fa-fw fa-plus-circle" />
+          </b-button>
+        </a>
+        <template v-if="visibleTab === 'overviews'">
+          <!-- link group create form -->
```
Comment (awick) [inline cont3xt/vueapp/src/components/pages/Settings.vue:47]: wrong comment text?

File: cont3xt/vueapp/src/components/integrations/DefaultCard.vue
Comment (awick) [inline cont3xt/vueapp/src/components/integrations/DefaultCard.vue:1]: lets not call it defaultcard

File: cont3xt/vueapp/src/utils/InputDropdown.vue [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      <b-form-input
+          ref="integrationViewDropdownSearch"
+          @keydown.enter="selectTopOption"
+          @focusin="setBarFocused"
+          @focusout="setBarUnfocused"
+          v-model="filter"
+      />
+    </b-input-group>
+
+<!--    <b-dropdown-item v-for="option in options">-->
```
Comment (31453) [inline cont3xt/vueapp/src/utils/InputDropdown.vue:16]: what's this for?

Comment (TobySalusky) [reply cont3xt/vueapp/src/utils/InputDropdown.vue]: WIP from auto-complete for overview fields, I've added a todo

**Review by 31453 — APPROVED** [review]
LGTM ✨

**Pull request merged**